sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/TruncateTableSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.command.v2
+
+import org.apache.spark.sql.AnalysisException
+import org.apache.spark.sql.execution.command
+
+/**
+ * The class contains tests for the `TRUNCATE TABLE` command to check V2 table catalogs.
+ */
+class TruncateTableSuite extends command.TruncateTableSuiteBase with CommandSuiteBase {
+
+  // TODO(SPARK-34290): Support v2 TRUNCATE TABLE
```

Comment:
The v2 API is ready, and this should be easy to do?

Reply:
Not yet, this https://github.com/apache/spark/pull/31420 is the first step.